pkgs/games/maptool/default.nix

```diff
@@ -99,15 +97,20 @@ stdenvNoCC.mkDerivation {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/bin
-    makeWrapper ${jre}/bin/java $out/bin/${binName} \
-      "''${gappsWrapperArgs[@]}" \
-      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ ffmpeg ]} \
-      --add-flags '${lib.concatStringsSep " " jvmArgs} net.rptools.maptool.client.LaunchInstructions'
+    dest=$out/${appClasspath}
+    install -dm755 "$dest"
+    bsdtar -xf "$src" -C "$dest" --strip-components 4 opt/maptool/lib/app/{'*.jar',readme}
```

Comment:
The code would be easier to understand if we didn't variablize dest

Reply:
I'd agree if I didn't need to use `dest` twice, once to create it and once to copy things into it. Each time it's defined, it's used only in the following two lines; this really isn't much of a barrier to understanding.